app/code/Magento/PricingStorefront/Model/PriceRepository.php

```diff
@@ -0,0 +1,238 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\PricingStorefront\Model;
```

Comment:
IT's better to distinguish business logic and work with the data retreiving  and create one more namespace for repository classes